src/pages/login/login.ts

```diff
@@ -86,67 +86,47 @@ export class LoginPage {
               self.cacheService.setLocal('timelineID', data.Timelines[0].Timeline.id);
               self.cacheService.setLocal('teams', data.Teams);
               self.cacheService.setLocal('gotNewItems', false);
-              // get game_id data after login
-              this.gameService.getGames()
-                  .subscribe(data => {
-                    console.log("game data: ", data);
-                    if (data && data.Games) {
-                      data.Games.map(game => {
-                        console.log("game id: ", game.id);
-                        if (game && game.id) { // avoid storing empty game id
-                          this.cacheService.setLocal('game_id', game.id);
-                        }
+              let getGame = this.gameService.getGames();
+              let getUser = this.authService.getUser();
+              let getMilestone = this.milestoneService.getMilestones();
+              Observable.forkJoin([getGame, getUser, getMilestone])
+                .subscribe(
+                  results => {
+                    loading.dismiss().then(() => {
+                      // results[0] game API data
+                      this.gameID = results[0].Games[0].id;
```

Comment:
@DamengRandom i believe you branched out from [ISDK-81](https://github.com/intersective/practera-sdk-ionic/pull/45), maybe you can start by looking into the fixing branch first.

Reply:
Yup sure @trtshen, I will merge my own branch first and make it consistent.